Extensions/TileMap/tilemapruntimeobject.js

```diff
@@ -36,8 +39,18 @@ gdjs.TileMapRuntimeObject.prototype.getRendererObject = function() {
   return this._renderer.getRendererObject();
 };
 
-gdjs.TileMapRuntimeObject.prototype._updateAnimationFrame = function() {
-  console.log("ANIMATTTEE") // not updating :(
+gdjs.TileMapRuntimeObject.prototype.update = function(runtimeScene) {
+  var elapsedTime = this.getElapsedTime(runtimeScene) / 1000; 
+  
+  this._frameElapsedTime += this._animationPaused ? 0 : elapsedTime * this._animationSpeedScale; 
+  console.log(this._frameElapsedTime);
+  // 0.25 = 4 fps - todo expose fps - calculate fps like in IDE?
+  if ( this._frameElapsedTime > 0.25 ) {
```

Comment:
Should be a `while` - you can have more than one frame increment if your game is too slow to update or your animation is too fast - or both.

Reply:
do you mean instead of an if statement, this should be a while loop? Ok :)

Reply:
Yes - if for example 50 milliseconds happened since the last frame (i.e: `getElapsedTime` will return 50) (because your game slowed down to 20fps because of something) and your object is animating at 60fps (i.e: a frame every 1/60 = 0.016s = 16 milliseconds), then you must pass to the third frame after the current frame (because 50-16-16-16) and still leave 2 ms in the counter of the elapsed time.

You may see that we can implement this with the modulo operator, but let's see first if it's worth doing it because the animations in pixi-tilemap seem global.